src/scene/vcFlythrough.cpp

```diff
@@ -274,7 +283,7 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
       for (size_t i = 0; i < udLengthOf(vcScreenshotResolutions); ++i)
       {
         if (ImGui::Selectable(vcString::Get(vcScreenshotResolutionNameKeys[i])))
-          m_selectedResolutionIndex = (int)i;
+          udProjectNode_SetMetadataInt(m_pNode, "selectedResolutionIndex", (int32_t)i);
```

Comment:
Doesn't really matter since this shouldn't be saved but you shouldn't save the index

Reply:
so "m_selectedResolutionIndex = (int)i;" is fine?